src/xdocs/checks/naming/packagename.xml

```diff
@@ -58,13 +58,10 @@
         </source>
         <p id="Example1-code">Code Example:</p>
         <source>
-package com; // OK
-package COM; // violation, name 'COM' must match pattern '^[a-z]+(\.[a-zA-Z_]\w*)*$'
-package com.checkstyle.checks; // OK
-package com.A.checkstyle.checks; // OK
-package com.checkstyle1.checks; // OK
-package com.checkSTYLE.checks; // OK
-package com._checkstyle.checks_; // OK
+package com.puppycrawl.tools.checkstyle.checks.naming.packagename;
+
+public class Example1 {
+}
         </source>
         <p id="Example2-config">
           An example of how to configure the check to ensure with packages start with a lowercase
```

Comment:
uppercase

Reply:
sorry what do you mean

Reply:
Change words to
`An example of how to configure the check to ensure with packages start with a uppercase`

Reply:
still not done